docs/changes/2600.feature.rst

```diff
@@ -0,0 +1,2 @@
+Add Interpolator class to generalize the PointingInterpolator in the io collection.
+Add new interpolator class called StepInterpolator to be used for calibration data.
```

Comment:
maybe remove the mention of StepInterpolator now that it has moved (otherwise this will end up in the release notes).

Reply:
yes, i will do that